pygotham/frontend/templates/talks/proposal.html

```diff
@@ -13,6 +13,9 @@
 
         <div class="row">
           {{ wtf.horizontal_field(form.name, placeholder='Title') }}
+          <small>
+            Got a Co-Presenter? <a href="mailto:organizers@pygotham.org" >Let us know!</a>
```

Comment:
You are right, we could have that approach. I think I was a bit unclear by the discussion in the ticket. 

If we do not have this approach, we will indeed require all co-presenters to have accounts. Is it alright for we to show presenters other usernames? could this influence people's decision making? IE: I don't want to talk about sqlalchemy because mike bayer is going to be there.